pkg/crc/telemetry/telemetry.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os/user"
 	"strings"
 	"sync"
+
```

Comment:
The main thing this commit achieves is that it replaces `/home/XXXXX` in telemetry errors with `$HOME`, and same for `XXXXX` as a username replacement which is replaced with `$USERNAME`.
The homedir was already sanitized before being sent to telemetry through username replacement.